client/src/components/TimetableSwitcher/TimetableSwitcher.tsx

```diff
@@ -16,6 +18,7 @@ import {
 	renameTimetable,
 	copyTimetable,
 } from '../../reducers/timetable';
+import selectFilteredTimetable from '../../selectors/timetable';
```

Comment:
The export for this was updated recently. You might have to do `{ selectFilteredTimetable }`. Please check.
```suggestion
import { selectFilteredTimetable } from '../../selectors/timetable';
```